docs/part-4-labeling-the-data-and-retrain/introduction.md

```diff
@@ -6,6 +6,11 @@ title: "Part 4 - Introduction"
 
 Learn how to label new data and retrain the model using Label Studio.
 
+!!! note
+
+     Although in previous parts we transitioned from local to cloud-based services,
+     this part will focus on labeling data locally for the sake of simplicity.
```

Comment:
Remove "for the sake of simplicity". It make it sound like labelling locally isn't viable as a solution and raises more question. We might also expand later on remote LabelStudio labelling.